udfs/flex/collections/zip.md

```
@@ -0,0 +1,73 @@
+# coll.zip
+
+## Description
+Combines two lists element-by-element into a list of pairs. Each pair contains one element from the first list and the corresponding element from the second list. The resulting list has the length of the shorter input list.
+
+## Syntax
+```cypher
+flex.coll.zip(list1, list2)
+```
+
+## Parameters
+| Parameter | Type | Required | Description |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)